docs/eample-findBy*.md

```diff
@@ -0,0 +1,142 @@
+---
+id: example-findBy*
```

Comment:
Note: I am not sure if we want to add this file. But if we do, can we avoid using non-url safe characters like `*`, and fix the filename?

Reply:
Done @alexkrolick! I used `*` as I was trying to follow the convention in the docs but since this example file only contains a `findByText` example, I've renamed it to `example-findByText`.

I've also added this new example to the site navigation.